core/src/main/java/fj/data/TreeZipper.java

```diff
@@ -627,6 +627,9 @@ else for (final TreeZipper<A> loc : parent())
             }, f.f(this));
   }
 
+  /**
+   * @param <A>

```

Comment:
This might be triggering the Javadoc fail
```
javadoc: error - An internal exception has occurred. 
	(com.sun.tools.javac.code.ClassFinder$BadClassFile: bad class file: /home/travis/build/functionaljava/functionaljava/core/build/libs/functionaljava-5.0-SNAPSHOT.jar(/fj/data/TreeZipper$2.class)
  undeclared type variable: A
```

Reply:
ah I added that in hope to fixe the issue. But that was it. Will remove.